core-build/gulp-core-build/src/tasks/JestTask.ts

```diff
@@ -152,7 +152,26 @@ export class JestTask extends GulpTask<IJestConfig> {
     const pattern: string = path.join(srcRoot, '**/__snapshots__/*.snap');
     globby.sync(pattern).forEach(sourceFile => {
       const destination: string = sourceFile.replace(srcRoot, destRoot);
-      fsx.copySync(sourceFile, destination);
+      const jsDestination: string = destination.replace(/\.test\..+\.snap$/, '.test.js.snap');
+      const jsxDestination: string = destination.replace(/\.test\..+\.snap$/, '.test.jsx.snap');
+      const tsDestination: string = destination.replace(/\.test\..+\.snap$/, '.test.ts.snap');
+      const tsxDestination: string = destination.replace(/\.test\..+\.snap$/, '.test.tsx.snap');
```

Comment:
If you are using jest directly, you would not need to copy those files around. Those should be created at the desired location.

Reply:
@qz2017 This is not correct. Jest will complaint there there is out-of-dated snap file `abc.test.js.snap` then create a new one with name `abc.test.ts.snap` for me. However, I cannot check-in the `ts` one because it is breaking the `gulp test` build.